test/e2e_node/services/services.go

```diff
@@ -43,6 +43,14 @@ import (
 // TODO(random-liu): Move this file to a separate package.
 var serverStartTimeout = flag.Duration("server-start-timeout", time.Second*120, "Time to wait for each server to become healthy.")
 
+// TODO(random-liu): Move kubelet start logic out of the test.
+// TODO(random-liu): Move log fetch logic out of the test.
+// There are different ways to start kubelet (systemd, initd, docker, rkt, manually started etc.)
+// and manage logs (journald, upstart etc.).
+// For different situation we need to mount different things into the container, run different commands.
+// It is hard and unnecessary to deal with the complexity inside the test suite.
+var containerize = flag.Bool("containerize", false, "If true, the test suite will not start kubelet, and fetch system log (kernel, docker, kubelet log etc.) to the report directory.")
```

Comment:
containerize is not an appropriate flag name here. How about `conformance` instead?